Credit memo refunds (#409)
magento/merchdocs: src/sales/credit-memo-create.md

---
title: Create a Credit Memo
---

Before a credit memo can be printed, it must first be generated for an invoiced order. You can issue both online and offline refunds from an open credit memo, depending on the payment method. <!--{% if "Default.EE-B2B" contains site.edition %}-->Refunds can also be applied to store <!--{% if "Default.B2B Only" contains site.edition %}--> or company <!--{% endif %}-->credit.<!--{% endif %}-->

- Purchases made by credit card can be refunded either online or offline.
- Purchases made by check or money order must be refunded offline.

Any credit memo with a [status]({% link sales/order-status.md %}) of “open” has an outstanding refund due. The refund workflow for orders paid by credit card is determined by the Payment Action setting in the configuration of the payment method. Refunds cannot be issued until the transaction is settled.

- If Payment Action is set to `Authorize`, you must first generate the invoice from the Magento Admin before a credit memo can be created.

> - If Payment Action is set to `Authorize and Capture`, the invoice has already been  generated by the payment processor, but the funds aren't available until the transaction is settled. This brief waiting period is recommended by many payment processors as a security measure, and can usually be handled automatically.Transactions can also be settled manually from your merchant account with the payment processor.

Reviewer: ```suggestion
- If Payment Action is set to `Authorize and Capture`, the invoice has already been generated by the payment processor, but the funds aren't available until the transaction is settled. This brief waiting period is recommended by many payment processors as a security measure, and can usually be handled automatically. Transactions can also be settled manually from your merchant account with the payment processor.
```

Reviewer: A couple of spacing issues here:
- There is an extra space between "been  generated"
- There is no space between "automatically.Transactions"

{:.bs-callout-info}
If using [Vertex Cloud]({% link tax/vertex.md %}), see the [Vertex Settings]({% link configuration/sales/tax.md %}) configuration section for information about issuing refunds for sales orders that are invoiced when the status is either `Suspected Fraud` or `Canceled`.

## Create a credit memo

Determine the type of refund that you want to issue. Then, use one of the following methods to generate the credit memo and issue a refund.

![]({% link images/images/order-credit-memo.png %}){: .zoom}
_Create Credit Memo_

### Method 1: Issue a refund for a credit purchase

1. On the _Admin_ sidebar, click **Sales** > **Orders**.

1. Find the order in the grid, and click **View**.

   <!--{% if "Default.CE Only" contains site.edition %}-->
   ![]({% link images/images/orders-grid.png %}){: .zoom}
   <!--{% endif %}-->
   <!--{% if "Default.EE-B2B" contains site.edition %}-->
   ![]({% link images/images-ee/orders-grid.png %}){: .zoom}
   <!--{% endif %}-->
   _Orders Grid_

1. If the Credit Memo button is visible in the button bar, do one of the following:

   - To issue an `offline` refund, go to step #6.
   - To issue an `online` refund, continue with step #4.

> 1. Click **Invoices** in the panel on the left. Find the invoice in the grid and click **View**.

Reviewer 2: ```suggestion
1. Click **Invoices** in the left panel.

1. Find the invoice in the grid and click **View**.
```

   ![]({% link images/images/order-invoices-grid.png %}){: .zoom}
   _Invoices Grid_

1. Scroll down to the **Order Total** section of the invoice, and verify that the invoice is set to `Capture Online`. Then, click <span class="btn">Submit Invoice</span>.

   ![]({% link images/images/order-invoice-capture-online.png %}){: .zoom}
   _Capture Online_

> 1. In the button bar at the top of the invoice, click **Credit Memo**. Then, verify the information in the **Items to Refund** section. If applicable, do the following:

Reviewer 2: ```suggestion
1. In the button bar at the top of the invoice, click **Credit Memo**. Then, verify the information in the **Items to Refund** section. If applicable, do the following:
```
```suggestion
1. In the button bar at the top of the invoice, click **Credit Memo**.

1. Verify the information in the **Items to Refund** section and do the following, if applicable:
```

   - To return the product to inventory, select the **Return to Stock** checkbox.

      The product returns to stock automatically if [Product Stock Options]({% link configuration/catalog/inventory.md %}) is set to `Automatically Return Credit Memo Item to Stock`. With Magento Inventory Management enabled, the item returns to the source that sent the shipment.

   - Update the **Qty to Refund**, and click <span class="btn">Update Qty's</span>.

      ![]({% link images/images/invoice-credit-memo-items-to-refund.png %}){: .zoom}

>       _Intems to Refund_

Reviewer: ```suggestion
      _Items to Refund_
```

1. To add a comment, enter the text in the **Credit Memo Comments** box.

   - To send an email notification to the customer, mark the **Email Copy of Credit Memo** checkbox.

1. Do the following, as applicable:

      <!--{% if "Default.EE-B2B" contains site.edition %}-->
      - To refund the amount to the customer's store credit, select the **Refund to Store Credit** checkbox.

      <!--{% endif %}-->
      <!--{% if "Default.B2B Only" contains site.edition %}-->
      - To refund the amount to the customer's company credit, select the **Refund to Company Credit** checkbox.

      <!--{% endif %}-->
      - To issue on offline refund, click <span class="btn">Refund Offline</span>.
      - To issue an online refund, click <span class="btn">Refund</span>.
      <!--{% if "Default.B2B Only" contains site.edition %}-->
      - If the purchase was paid with company credit, click <span class="btn">Refund to Company Credit</span>.
      <!--{% endif %}-->

      ![]({% link images/images/credit-memo-order-total-refund.png %}){: .zoom}
      _Items to Refund_

### Method 2: Issue an offline refund for check or money order

1. On the _Admin_ sidebar, click **Sales** > **Orders**.

> 1. Find the completed order in the grid. Then click the **View** link to open the order.

Reviewer 2: ```suggestion
1. Find the completed order in the grid and click the **View** link to open it.
```

> 1. In the button bar at the top of the page, click **Invoice**. Scroll down to the bottom of the page and click <span class="btn">Submit Invoice"</span>.

Reviewer 2: ```suggestion
1. In the button bar at the top of the page, click **Invoice**. Scroll down to the bottom of the page and click <span class="btn">Submit Invoice"</span>.
```
```suggestion
1. In the button bar at the top of the page, click **Invoice**.

1. Scroll down to the bottom of the page and click <span class="btn">Submit Invoice"</span>.
```

1. In the button bar at the top of the invoice, click **Credit Memo**.

   <!--{% if "Default.CE Only" contains site.edition %}-->
   ![]({% link images/images/order-create-credit-memo.png %}){: .zoom}
   <!--{% endif %}-->
   <!--{% if "Default.EE Only" contains site.edition %}-->
   ![]({% link images/images/order-create-credit-memo.png %}){: .zoom}
   <!--{% endif %}-->
   <!--{% if "Default.B2B Only" contains site.edition %}-->
   ![]({% link images/images-b2b/order-account-info-credit-memo.png %}){: .zoom}
   <!--{% endif %}-->
   _Create Credit Memo_

> 1. Verify the information in the **Items to Refund** section, and if applicable, do the following:

Reviewer 2: ```suggestion
1. Verify the information in the **Items to Refund** section and do the following, if applicable:
```

   - Select the **Return to Stock** checkbox if you want to return the returned product to inventory.

>       With Magento Inventory Management enabled, the inventory quantity returns to the source that sent the shipment. The product returns to stock automaticallys if [Product Stock Options]({% link configuration/catalog/inventory.md %}) is set to `Automatically Return Credit Memo Item to Stock`.

Reviewer: ```suggestion
      With Magento Inventory Management enabled, the inventory quantity returns to the source that sent the shipment. The product returns to stock automatically if [Product Stock Options]({% link configuration/catalog/inventory.md %}) is set to `Automatically Return Credit Memo Item to Stock`.
```

>    - Update the **Qty to Refund**, and click <span class="btn">Update Qty's</span>.

Reviewer 2: ```suggestion
   - Update the **Qty to Refund** and click <span class="btn">Update Qty's</span>.
```

      The amount to be credited cannot exceed the maximum amount that is available for refund.

   ![]({% link images/images/credit-memo-items-to-refund.png %}){: .zoom}
   _Items to Refund_

1. Update the **Refunds Totals** section as applicable:

   - In the **Refund Shipping** field, enter any amount that is to be refunded from the shipping fee.

      This field initially displays the total shipping amount from the order that is available for refund. It is equal to the full shipping amount from the order, less any shipping amount that has already been refunded. Like the quantity, the amount can be reduced, but not increased.

   - In the **Adjustment Refund** field, enter a value to be added to the total amount refunded as an additional refund that does not apply to any particular part of the order (shipping, items, or tax).

      The amount entered cannot raise the total refund higher than the paid amount.

   - In the **Adjustment Fee** field, enter a value to be subtracted from the total amount refunded.

      This amount is not subtracted from a specific section of the order such as shipping, items, or tax.

>    - If the purchase was paid with store credit, mark the **Refund to Store Credit** checkbox to credit the amount to the customer account balance.

Reviewer 2: ```suggestion
   - If the purchase was paid with store credit, select the **Refund to Store Credit** checkbox to credit the amount to the customer account balance.
```

> 1. To add a comment, enter the text in the **Credit Memo Comments** box. Then, do the following:

Reviewer 2: ```suggestion
1. To add a comment, enter the text in the **Credit Memo Comments** box and do the following:
```

>    - To send an email notification to the customer, mark the **Email Copy of Credit Memo** checkbox.

Reviewer 2: ```suggestion
   - To send an email notification to the customer, select the **Email Copy of Credit Memo** checkbox.
```

>    - To include the comments you have entered in the email, mark the **Append Comments** checkbox.

Reviewer 2: ```suggestion
   - To include the comments you have entered in the email, select the **Append Comments** checkbox.
```
